src/site/content/en/blog/prefers-color-scheme/index.md

```diff
@@ -456,7 +456,7 @@ so that they never compete with resources that are needed by the site right now.
   <figcaption class="w-figcaption">Fig. — Site in default light mode on a browser that doesn’t support <code>prefers-color-scheme</code> loads the dark mode CSS with lowest priority.</figcaption>
 </figure>
 
-### Finding out if dark mode is supported
+### Finding out if dark mode is supported by the browser
```

Comment:
@emilio via https://github.com/filamentgroup/loadCSS/issues/304#issuecomment-505229524:
> @tomayac fwiw, that post contains an error in the "Finding out if dark mode is supported" section. (prefers-color-scheme) is supposed not to match if there's no preference for light or dark, so the right way to test whether it's supported is not window.matchMedia('(prefers-color-scheme)').matches but window.matchMedia('(prefers-color-scheme)').matches || window.matchMedia('(prefers-color-scheme: no-preference)').matches (or something like that). See w3c/csswg-drafts#3857 for more fun stuff related to that.

Thanks for the pointer, but this is actually really about finding out whether the `prefers-color-scheme` is supported (i.e., understood) by the browser at all. So the current check is correct. I have appended "by the browser" to hopefully make this clearer.

Reply:
My point is that the browser may understand the `prefers-color-scheme` media feature while returning false for `matchMedia('(prefers-color-scheme)').matches`. The way to test whether the media query is correctly parsed is either what I mentioned above or `machMedia('(prefers-color-scheme)').media != 'not all'`.

Reply:
Oooh, I see what you mean now. The boolean context… Thanks a lot! 

[L4](https://www.w3.org/TR/mediaqueries-4/#mq-boolean-context) had this:
*When written like this, the media feature is evaluated in a boolean context. If the feature would be true for any value other than the number 0, a <dimension> with the value 0, or the keyword none, the media feature evaluates to true. Otherwise, it evaluates to false.*

The **addition** in [L5](https://drafts.csswg.org/mediaqueries-5/#boolean-context) is important:
*When written like this, the media feature is evaluated in a boolean context. If the feature would be true for any value other than the number 0, a <dimension> with the value 0, the keyword none, **or a value explicitly defined by that media feature to evaluate as false in a boolean context**, the media feature evaluates to true. Otherwise, it evaluates to false.*

And `no-preference` is [well defined](https://drafts.csswg.org/mediaqueries-5/#valdef-media-prefers-color-scheme-no-preference) as such a value. I will correct the article and credit you :-) Thanks again!